plugins/woocommerce-admin/client/products/categories/product-details-category.tsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { TextControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { ProductCategoryLayout } from '../layout/product-category-layout';
+import { ProductFieldLayout } from '../layout/product-field-layout';
+
+export const ProductDetailsCategory: React.FC = () => {
+	return (
+		<ProductCategoryLayout
+			title={ __( 'Product details', 'woocommerce' ) }
+			description={ __(
+				'This info will be displayed on the product page, category pages, social media, and search results.',
+				'woocommerce'
+			) }
+		>
+			<ProductFieldLayout name="Name">
```

Comment:
Is the `name` prop unique globally, per category, or per something else? How is this enforced?

Reply:
Good point, I will add `categoryName` as an additional prop, as we probably won't have the same category name and field name.